web/js/map-wmts-buckinghamshire.js

```diff
@@ -137,11 +137,9 @@ fixmystreet.maps.config = function() {
 
 fixmystreet.maps.marker_size = function() {
     var zoom = fixmystreet.map.getZoom() + fixmystreet.zoomOffset;
-    if (zoom >= 7) {
+    if (zoom >= 1) {
```

Comment:
Similarly here, presumably. Could leave 1M in, leave these alone and only change min/max_resolution? :)

Reply:
yes, good point. done now.